src/routes/docs/products/functions/executions/+page.markdoc

```diff
@@ -17,25 +17,25 @@ Here's the information shown on this table.
 - Column
 - Description
 ---
-- EXECUTION ID
+- `Execution id`
```

Comment:
It's not code so it shouldn't be formatted as code

Reply:
@Meldiron what are your thoughts on this? I know you requested this change for these keys in the table to be in code syntax format.

Reply:
Im fine with sentence case. I just didn't want it to be uppercase - its only uppercase in Console for design reasons.